subsys/bluetooth/mesh/gen_lvl_srv.c

```diff
@@ -77,7 +77,9 @@ static int set(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
 		rsp_status(model, ctx, &status);
```

Comment:
Responding to old transactions is common for most models where applicable. Im not 100% sure why, but I guess it prevents blocking client calls from blocking until timeout when the message is actually received by the server. There might be other good reasons as well, since it is clearly done on purpose: https://github.com/nrfconnect/sdk-nrf/blob/df5ecde63f9a13710415971313d2c299b5d2f265/subsys/bluetooth/mesh/gen_onoff_srv.c#L73-L75

Ideally we should do as done in generic onoff set CB. However, it will complicate things since we have other models that extends this model and is dependent on the current behavior. I saw this as the least invacive fix